infrastructure/quick-deploy/localhost/parameters.tfvars

```diff
@@ -213,6 +213,58 @@ compute_plane = {
       ]
     }
   },
+  # Partition for the htcmock worker with tcp socket
```

Comment:
I'm not sure this requires a new partition...

Reply:
Ok, htcmock partition uses tcp now.